src/tags/views.py

```diff
@@ -58,7 +58,7 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(**kwargs)
 
 
-class TagDetailStudent(TagDetailMixin, NonPublicOnlyViewMixin, LoginRequiredMixin, DetailView):
+class TagDetailStudent(TagDetailMixin):
```

Comment:
Mixin is something to "mix in" with a base class to give it extra properties and methods, but here it is being used as a base class it seems?  i.e to add some stuff or set some values for the base class of DetailView.

Might just mean that TagDetail Mixin ISNT a mixin (most of this is just naming conventions, but confusion when not followed, and coders intention is hard to read when conventions are followed=)

Reply:
removed mixin from class name